src/types/types.js

```diff
@@ -630,6 +630,9 @@ export type EdgeCurrencyEngine = {
     paymentProtocolUrl: string
   ) => Promise<EdgePaymentProtocolInfo>,
 
+  // Accelerating:
+  +accelerate: (tx: EdgeTransaction) => Promise<EdgeTransaction | null>,
```

Comment:
This should be `accelerate?:`, since most engines won't have it implemented yet.